entrypoint.sh

```diff
@@ -22,7 +22,7 @@ echo "Setting up git machine..."
 git_setup
 
 echo "Forcing tag update..."
-git tag -a latest -m "This tag has been auto-generated by latest-tag" "${GITHUB_SHA}" -f
+git tag -a latest -m $INPUT_INTRODUCTION "${GITHUB_SHA}" -f
```

Comment:
This is linked to one of the previous: do you want it to be named `description` or `introduction`?